packages/coil-extension/manifest.json

```diff
@@ -31,5 +31,5 @@
       "run_at": "document_start"
     }
   ],
-  "permissions": [ "activeTab", "storage", "notifications", "tabs", "<all_urls>" ]
+  "permissions": [ "webNavigation", "activeTab", "storage", "notifications", "tabs", "<all_urls>" ]
```

Comment:
This is going to require extra permissions

Reply:
Nothing more revealing (in terms of privacy) than what we already require but it could be off-putting to some users.